### ISSUE TITLE
Update meep on Pypi?

### ISSUE THREAD

**Issue by jaredmagnusson** [NanoComp/meep#2699]
Is there a way to make the latest meep installable with pip, in order to avoid the headache of building from source or requiring conda? I see a version from 2018 on Pypi but nothing later. This would be for machines that already have python environments set up and where meep would be part of a larger workflow.

**Comment by oskooi** [comment]
I agree that PyPI provides a more natural workflow for working with Python packages than Conda which is more geared towards one-off installations. We did consider packaging the Meep binary using PyPI in 2018 but ended up choosing Conda partly because it had better support (at the time) for Meep's non-Python library dependencies including HDF5, harminv, and MPB.

We would consider switching to PyPI if someone could put together the package itself.

**Comment by zacoons** [comment]
+1 a PyPi package would be great
